cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -0,0 +1,450 @@
+name "Google Cloud SQL Idle Instance Recommender"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google Idle Cloud SQL Instance Recommendations and reports when it finds them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/google/cloud_sql_idle_instance_recommendations) and [docs.rightscale.com/policies](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
Still a slight change required - for the second link replace `[docs.rightscale.com/policies]` with `[docs.flexera.com/flexera/EN/Automation]`. Same for the CUD recommendation policy template